pkg/policy/api/l4.go

```diff
@@ -40,7 +42,7 @@ type PortProtocol struct {
 	// or "http-8080".
 	//
 	// +kubebuilder:validation:Pattern=`^(6553[0-5]|655[0-2][0-9]|65[0-4][0-9]{2}|6[0-4][0-9]{3}|[1-5][0-9]{4}|[0-9]{1,4})|([a-zA-Z0-9]-?)*[a-zA-Z](-?[a-zA-Z0-9])*$`
-	Port string `json:"port"`
+	Port string `json:"port,omitempty"`
```

Comment:
Should we add validation that `port` is not allowed when VRRP and IGMP are set?

Reply:
Yup, here is the validation - https://github.com/cilium/cilium/pull/39872/commits/d026da14ec0addf3e50d7a6f3c88607e37ebb74f#diff-cecfad25fc16b5cc6406ceabe9950e8ed4889e5a7ec9ab8d1f854ee5305e907fR654-R657.